manager/aica_django/converters/Knowledge.py

```diff
@@ -786,14 +813,17 @@ def nginx_accesslog_to_knowledge(
         )
     )
 
-    ipv4_addr = IPv4Address(log_dict["src_ip"])
-    ipv4_addr_knowledge = ipv4_addr.to_knowledge_node()
-    knowledge_nodes.append(ipv4_addr_knowledge)
+    if type(ipaddress.ip_address(log_dict["src_ip"])) is ipaddress.IPv4Address:
+        ip_addr_knowledge = IPv4Address(log_dict["src_ip"]).to_knowledge_node()
+    else:
```

Comment:
This should probably be elif, with a final else to handle the unexpected case where neither the IPv4 or IPv6 checks pass.

Reply:
This happens when no ip or hostname is logged in nginx of the remote host. This should not happen I suppose and I will throw a new Exception here. Or do you know a situation where this is normal ? In that case we should add a new KnowledgeNode and fill this information and use as knowledge.